mandatory/3-playing-computer.js

```diff
@@ -28,7 +28,7 @@ const f2 = function (a, b) {
 
```

Comment:
We need to anwser the questions in the comments section of this file (lines 9 to 15).

Reply:
I found that 'console.log(b)' caused an error.  I deleted and it fixed.

Reply:
Yes, this is question 1. We need also to write the answers to questions 3 to 7:
  3. What is printed to the console?
  4. How many times is "f1" called?
  5. How many times is "f2" called?
  6. What value does the "a" parameter take in the first "f1" call?
  7. What is the value of the "a" outer variable when "f1" is called for the first time?

Reply:
Actually I deleted it and run it. It passed it. Do I need to type some codes on it?